src/types/index.ts

```diff
@@ -60,13 +60,22 @@ export interface OperationObject {
   responses?: Record<string, ReferenceObject | ResponseObject>; // required
 }
 
+export interface ItemsObject {
```

Comment:
I believe we can remove this; this seems to be a duplication of `SchemaObject` but with fewer fields (see `SchemaObject["items"]` as a reference